pom.xml

```diff
@@ -137,4 +180,20 @@
             <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>hpi</goal>
+                        </goals>
+                        <phase>compile</phase>
```

Comment:
why? this doesn't seem right

Reply:
When I tried doing `mvn hpi:hpi` for the first time it always failed so I moved it to the compile phase so it would come after the code is compiled.

Reply:
can you raise an issue at https://github.com/jenkinsci/maven-hpi-plugin/

I don't understand that plugin well enough but don't really want this here, I've never had the need to call `mvn hpi:hpi` directly.

Either:
`mvn clean package` or `mvn hpi:run` should be sufficient and work fine

Reply:
OK, I've removed that section from the pom.xml and adjusted the ./deploy.sh file.